cl/opinion_page/tests.py

```diff
@@ -210,3 +218,132 @@ def test_does_the_sitemap_have_content(self):
         # Class attributes are set, just run the test in super.
         self.expected_item_count = self.get_expected_item_count()
         super(OpinionSitemapTest, self).does_the_sitemap_have_content()
+
+
+class UploadPublication(TestCase):
+
+    fixtures = ["tenn_judges.json"]
```

Comment:
Let's make a minimal fixture for this with only one or two judges instead of importing so many judges (which takes more time during tests).

Reply:
Need at a minimum four judges in the fixture to test our tests.  But noted.  I removed 13 of the 17